File: 2195-time-needed-to-buy-tickets/time-needed-to-buy-tickets.cpp
class Solution {
public:
    int timeRequiredToBuy(vector<int>& tickets, int k) {
        int count=1;
        int len=tickets.size();
        for(int i=0;i<k;i++){
            count+=min(tickets[i],tickets[k]);
        }
        for(int i=k;i<len;i++)
        {
            count+=min(tickets[k]-1,tickets[i]);
        }
        return count;
    }
};

File: 2195-time-needed-to-buy-tickets/time-needed-to-buy-tickets.java
class Solution {
    public int timeRequiredToBuy(int[] tickets, int k) {
        int time = 0;
        int len = tickets.length;
        for(int i=0;i<=k;i++){
            time+=Math.min(tickets[k],tickets[i]);
        }
        for(int i=k+1;i<len;i++){
            time+=Math.min(tickets[k]-1,tickets[i]);
        }
        return time;
    }
}